src/plugin/timeline.js

```diff
@@ -16,7 +16,9 @@
  * performance for large files
  * @property {string} fontFamily='Arial'
  * @property {number} fontSize=10 Font size of labels in pixels
- * @property {function} formatTimeCallback=→00:00
+ * @property {number} startSeconds=0 Start time in second for begining timeline rule
```

Comment:
there are 2 typos: `second` and `begining`

Reply:
OK